src/main/java/frc/robot/subsystems/intake/IntakeIOSim.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix6.controls.PositionVoltage;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import lib.motors.SparkMaxSim;
+import lib.motors.TalonFXSim;
+
+public class IntakeIOSim implements IntakeIO {
+    private TalonFXSim angleMotor;
+    private SparkMaxSim frontRoller;
+    private SparkMaxSim centerRoller;
+    private PositionVoltage positionRequest = new PositionVoltage(0);
+
+    public IntakeIOSim(PIDController angleController) {
+        angleMotor = new TalonFXSim(1, IntakeConstants.GEAR_RATIO, 0.3);
+        frontRoller = new SparkMaxSim(1, 1, 0);
+        centerRoller = new SparkMaxSim(1, 1, 0);
```

Comment:
Oh goodness. This is a perfect recipe for an exception. If the moment of inertia is zero, the rotational acceleration would be the moment (force times radius) divided by zero.

Reply:
thanks I am good cook.

Reply:
423a178e32ada435b677b785a6c731fd229beeee